pages/about.vue

```diff
@@ -50,56 +51,74 @@
             alt=""
             format="webp"
             height="1278px"
+            loading="lazy"
             src="/img/about/ship-shape-sign.png"
             width="1278px"
           />
         </div>
       </div>
     </div>
 
-    <div class="section bg-white flex flex-wrap justify-center">
-      <div
-        class="
-          section-content
-          gap-20
-          grid grid-cols-1
-          items-center
-          lg:gap-32 lg:grid-cols-2
-        "
-        data-aos="fade-right"
-        data-aos-delay="400"
-      >
-        <div>
-          <h3>Fun facts about us</h3>
-          <ul class="navy-bullets">
-            <li>
-              We consistently contribute to the
-              <nuxt-link to="/open-source">open-source community</nuxt-link>.
-            </li>
-            <li>
-              We run a
-              <a href="https://ecologi.com/ship-shape">climate positive</a>
-              shop.
-            </li>
-            <li>
-              We started a
-              <a href="https://www.whiskeywebandwhatnot.fm/"
-                >podcast for developers</a
-              >
-              to share our love for whiskey, web development, and whatnot.
-            </li>
-            <li>We have team members in 8+ locations across North America.</li>
-          </ul>
+    <div class="section flex flex-wrap justify-center bg-white">
+      <div class="section-content" data-aos="fade-right" data-aos-delay="400">
+        <h3 class="text-5xl">Fun facts about us</h3>
+
+        <div class="grid grid-cols-1 gap-5 lg:grid-cols-3">
+          <div class="bg-blue-light p-5 rounded-md lg:p-12">
+            <nuxt-img
+              class="h-auto m-auto my-4 rounded-lg w-32 lg:w-44"
+              format="webp"
+              height="336"
+              src="/img/hero-images/www.png"
+              width="336"
+            />
+
+            We started a
+            <a href="https://www.whiskeywebandwhatnot.fm/"
+              >podcast for developers</a
+            >
+            to share our love for whiskey, web development, and whatnot.
+          </div>
+          <div class="bg-blue-light p-5 rounded-md lg:p-12">
+            <a
+              href="https://ecologi.com/ship-shape?r=60427b02d9eb6b001d969215"
```

Comment:
@hannakim91 I recommend you start with the changes I had here already and use them for the fun facts section. This Ecologi image is supposed to be dynamic and should use this code and the other ones seem blurry, so you may want to borrow styles and things from here.

Reply:
Switched those back and added a couple styles to align the ecologi card - it was behaving differently b/c it's an `a` for image